adr/2024_01_31_drawing_icon_size.md

```diff
@@ -0,0 +1,102 @@
+- small
+- medium
+- big
+
+On the legacy viewer, the icons were served by [geoadmin/mf-chsdi3](https://github.com/geoadmin/mf-chsdi3) at a fix size of `48px` while on this viewer icons are served by [geoadmin/service-icons](https://github.com/geoadmin/service-icons) using a dynamic size depending on the selected UI size.
+
+Therefore we need to do some conversion between legacy and current viewer.
+
+Here below is a table about the size, the value for the new viewer are per `31.01.2024` and might change.
+
+### Label Size
+
+<!-- prettier-ignore-start -->
+| Size       | Legacy Scale | New Scale |
+| ---------- | ------------ | --------- |
+| very small | -            | 1         |
+| small      | 1            | 1.25      |
+| medium     | 1.5          | 1.5       |
+| big        | 2            | 2         |
+<!-- prettier-ignore-end -->
+
+### Icon Size
+
+<!-- prettier-ignore-start -->
+| Size       | Leg. Scale | Leg. Style Size | Leg. Icon Size* | Leg. Real Size | New Scale | New Style Size | New Icon Size* | New Icon size x scale | New Style size x scale |
+| ---------- | ---------- | --------------- | -------------- | -------------- | --------- | -------------- | ------------- | --------------------- | ---------------------- |
+| very small | -          | -               | -              | -              |      0.75 | 24             | 0.5x  (24px)  | 36                    | 18 |
+| small      | 0.5        | 48              | 24@2x (48px)   | 24             |     1.125 | 36             | 0.75x (36px)  | 81                    | 40.5 |
+| medium     | 0.75       | 48              | 24@2x (48px)   | 36             |       1.5 | 48             | 1x    (48px)  | 144                   | 72 |
+| big        | 1          | 48              | 24@2x (48px)   | 48             |      2.25 | 72             | 1.5x  (72px) | 324                   | 162 |
+<!-- prettier-ignore-end -->
+
+\* This is the size of the Icon return by the backend with the scale value asked in the backend.
+
+On the table above concerning the new viewer `scale` and `style size`, those values are automatically computed by openlayer the following way:
+
+- `style size` := size of the icon retrieved from the URL
+- `scale` := 1 (style scale default value)/ (32 / size)
+  See [/openlayers/openlayers/src/ol/format/KML.js#L2727](https://github.com/openlayers/openlayers/blob/7670a67fd013a0002bf84442e96d7399eb2403cc/src/ol/format/KML.js#L2727) and [openlayers/openlayers/src/ol/format/KML.js#L325](https://github.com/openlayers/openlayers/blob/7670a67fd013a0002bf84442e96d7399eb2403cc/src/ol/format/KML.js#L325)
+
+NOTE: The actual size of the icons on the backend `service-icons` is 96px but they are reduced to 48px for the scale 1x !
+
+## Descision
+
+Because the 4th UI size has been added to have bigger icons and not smaller icon it has been decided by Brice Schaffner and Pascal Barth to change the UI sizes as follow:
+
+- small => 24px
+- medium => 36px
+- big => 48px
+- bigger => 72px
```

Comment:
`large` instead of `bigger`? or `small` `medium` `large`, `xlarge`?

Reply:
Good idea

Reply:
@boecklic I updated the translation for the size see bottom of https://docs.google.com/spreadsheets/d/1bRzdX2zwN2VG7LWEdlscrP-wGlp7O46nvrXkQNnFvVY/edit#gid=54811248 or search for `extra_large`, what do you think ?